### DISCUSSION TITLE
Plots with shared axes

### DISCUSSION THREAD

**Discussion by LeafTeaNeko** [Kanaries/pygwalker#576] [Q&A] [answered]
Hi PyGwalker team, I'm trying to plot two variables next to each other. I tried the following this but it resulted in two plots instead of the variables plotted side by side. 
![image](https://github.com/Kanaries/pygwalker/assets/34652259/d172b7b8-5822-464f-a112-35da4f7bf50c)

 I am wondering if it is possible to plot multiple counts on a shared axis something like this:
![image](https://github.com/Kanaries/pygwalker/assets/34652259/274e0713-ecd6-47cd-b135-4f5c71d758d2)

Any ideas if that possible and if so, how do I go about doing that? Thanks in advance!

**Comment by longxiaofei** [comment, marked as answer]
Hi @LeafTeaNeko 

Using `measure names` and `measure values`

![image](https://github.com/Kanaries/pygwalker/assets/28337703/b41ab55c-4e4e-4bc0-9e1f-8d8e01e8fce3)

![image](https://github.com/Kanaries/pygwalker/assets/28337703/f2498c5e-c962-430a-90c7-c2b10eb9aba0)

**Reply by LeafTeaNeko** [reply]
Thank you very much Fei, this opens up a whole new world of analysis for me! Appreciate it very much

**Reply by jesseyu1984918** [reply]
Can you elaborate how this works, what does measure value and measure name mean?

**Reply by LeafTeaNeko** [reply]
@jesseyu1984918 Measure Values and Measure Names are two varaibles available in the list. They will be available by default and upon picking either, they will allow you to choose what those Measure values will be. This allows you to measure mulitple values on y-axis as the variable allows you to pick multiple data columns from your data. I recommend playing with them here ([demo](https://pygwalker-public-bucket.s3.amazonaws.com/demo.html)) because its hard to describe them without the context of a specific dataset. Have a peek around the documentation here: https://docs.kanaries.net/graphic-walker/data-viz/make-bar-chart
